### PR TITLE
Issue #13877: SpotBugs checks a map with containsKey(), before using …

### PR DESCRIPTION
**Pull request by AayushSaini101**
Related: https://github.com/checkstyle/checkstyle/issues/13877

### CODE REVIEW

File: src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java [hunk 1]
```diff
@@ -33,19 +36,31 @@ protected String getPackageLocation() {
...
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
+            "19:15: " + getCheckMessage(ParameterNumberCheck.class,
+                        ParameterNumberCheck.MSG_KEY, 7, 8),
 
         };
 
-        verifyWithInlineConfigParser(getPath("Example2.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example2.java"), expected);
```
Comment (romani) [inline src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java:64]: Why such changes are in this commit?

**Review by romani — CHANGES_REQUESTED** [review]
Looks like unrelated changes mixed in this PR

File: src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java (hunk 1 above)
Comment (AayushSaini101) [reply src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java]: Resolved @romani

**Review by romani — APPROVED** [review]
Ok to merge

File: src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java [hunk 2]
```diff
@@ -707,7 +708,6 @@ else if (SUPER_CLASS_PROPERTIES_JAVADOCS.containsKey(propertyName)
                     "Failed to find since version for %s", propertyName);
             throw new MacroExecutionException(message);
         }
-
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java:710]: Restore empty line.

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java (hunk 2 above)
Comment (AayushSaini101) [reply src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java]: Done @rnveach

**Comment by romani** [comment]
@AayushSaini101 , please recreate PR to let  circle CI sonar pass

**Pull request closed without merging**